.github/workflows/conformance.yml

```diff
@@ -0,0 +1,34 @@
+name: Ruby Conformance CI
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
Does this need to be 18.04 or can we use latest?

Reply:
Latest should be fine.